packages/@romejs/html-parser/index.ts

```diff
@@ -292,6 +323,27 @@ const createHTMLParser = createParser((ParserCore, ParserWithRequiredPath) =>
 			);
 		}
 
+		parseComment(): AnyComment {
+			const start = this.getPosition();
+			const token = this.expectToken("Comment");
+			const comment: AnyComment = {
+				value: token.value,
+				id: "",
+				type: "CommentBlock",
+			};
+
+			const finishedToken = this.finishNode(start, comment);
+
+			// TODO: to review this, it executes code inside JS parser
+			// new Error(`No comment found for id ${id}`);
+			// this.addComment({
+			// 	...comment,
+			// 	loc: finishedToken.loc
+			// });
+
+			return finishedToken;
```

Comment:
You can replace this with:

```suggestion
			this.comments.addComment({
				value: token.value,
				type: "CommentBlock",
				loc: this.finishLoc(start),
			});
```

`addComment` will finish the comment node, give it an `id`, push it to the relevant stacks, and then any `finishNode` calls will handle comment attachment.